src/components/chat-options/chat-options.ts

```diff
@@ -0,0 +1,72 @@
+import Block from '../../utils/Block';
+import template from './chat-options.template.hbs';
+import { Button } from '../button';
+import ChatsController from '../../utils/controllers/chatsController';
+import { ChatInput } from '../chat-input';
+import { logFormData } from '../../utils/logFormData';
+
+interface IChatOptions {
+  chatId: number;
+}
+
+export class ChatOptions extends Block<IChatOptions> {
+  constructor(props: IChatOptions) {
+    super(props);
+  }
+
+  protected initChildren() {
+    this.children.buttonRemoveChat = new Button({
+      text: 'Удалить чат',
+      events: {
+        click: () => {
+          ChatsController.deleteChat(this.props.chatId);
+        },
+      },
+    });
+
+    this.children.inputUserId = new ChatInput({
+      name: 'userId',
+      autocomplete: 'off',
+      className: 'user-id',
+      placeholder: 'Пользователь',
+    });
+
+    this.children.buttonAddUser = new Button({
+      text: 'Добавить пользователя',
+      events: {
+        click: (evt) => {
+          evt.preventDefault();
+          const data = logFormData('.add-remove-user');
+          if (data?.userId) {
+            ChatsController.addUser({
+              chatId: this.props.chatId,
+              users: [parseInt(data.userId, 10)],
+            });
+          }
+          (document.querySelector('.add-remove-user') as HTMLFormElement).reset();
```

Comment:
1. Если можно не использовать явное приведение типов, то лучше его не использовать
2. querySelector может вернуть null
```suggestion
          const form = document.querySelector<HTMLFormElement>('.add-remove-user')
          
          if (form)
            form.reset()
```